bentoml/_internal/configuration/default_configuration.yaml

```diff
@@ -38,6 +38,8 @@ api_server:
     metrics:
       host: 0.0.0.0
       port: 3001
+  runner_readiness_probe:
```

Comment:
We can just call this `readyz` maybe?

Reply:
I checked in with @sauyon and we'll keep this name more broad, like `health_probe` since the check isn't named `readyz` on the grpc side.